lbry/conf.py

```diff
@@ -461,8 +462,28 @@ def set_persisted(self, config_file_path=None):
         if self.persisted.upgrade():
             self.persisted.save()
 
+    @property
+    def needs_proactor(self):
+        major, minor, _ = platform.python_version_tuple()
+        if int(major) > 3 or (int(major) == 3 and int(minor) > 7):
+            return False
+        return platform.system() == "Windows"
```

Comment:
I don't think we need this. See later comment in `cli.py`.

Reply:
Fixed and pushed.